File: Leetcode/contains-duplicate.cpp
// https://leetcode.com/problems/contains-duplicate/
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int, int>mp;
        for(int i=0; i < nums.size(); i++)
        {
            mp[nums[i]]++;
            if(mp[nums[i]] > 1)
                return true; 
        }
        return false;      
    }
};

File: Leetcode/contains_duplicate.py
"""
Given an integer array nums, return true if any value 
appears at least twice in the array, 
and return false if every element is distinct.
"""
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        if len(nums) > len(set(nums)):
            return True
        return False